eclass-writing/text.xml

```diff
@@ -47,6 +47,17 @@ Roughly speaking, there are three kinds of eclass:
 <title>Adding and Updating Eclasses</title>
 <body>
 
+<note>
+Given that updating eclasses will invalidate the cache of all consumer ebuilds,
+it is good etiquette to ping other developers in e.g. <c>#gentoo-dev</c> on IRC
+or informally determine if there are other similar changes (mainly documentation)
+which should be pushed at the same time in order to avoid unnecessary cache
+regeneration within a few hours or days of each other.
+
+The same applies to your own work <d/> please prepare all of your commits and
+batch push them at once where possible.
+</note>
```

Comment:
This applies to the major eclasses only. For example, it is not a concern if I update elisp.eclass because ebuilds inheriting it typically don't inherit anything else.

Also, this note shouldn't go to the beginning of the section.

Reply:
do we have to come up with arbitrary thresholds now for deciding when something is "major" enough to cause needless cache churn? I fully agree with this and would like to have more devs abide by it.

Reply:
As I said, it makes little sense to coordinate updates of eclasses like elisp.eclass (or kernel-2.eclass, toolchain.eclass, texlive-module.eclass) which are pretty much orthogonal to other eclasses.

Reply:
Maybe I've misunderstood that note. Reading it again, it seems to apply to independent changes within _the same_ eclass, and then it makes sense of course.

Changing it to singular may help avoiding that ambiguity:
```suggestion
<note>
Given that updating an eclass will invalidate the cache of all consumer ebuilds,
it is good etiquette to ping other developers in e.g. <c>#gentoo-dev</c> on IRC
or informally determine if there are other similar changes (mainly documentation)
which should be pushed at the same time in order to avoid unnecessary cache
regeneration within a few hours or days of each other.

The same applies to your own work <d/> please prepare all of your commits and
batch push them at once where possible.
</note>
```

Reply:
Sounds good.